### ISSUE TITLE
Improve dashed arrows

### ISSUE THREAD

**Issue by dscso** [cetz-package/cetz#396]
maybe provide a simpler way to create dashed arrows than like that
```typst
line((0, 0), (0, 1), stroke: (thickness: .5pt, dash: "dashed"), mark: (end: ">", fill: black, stroke:black))
```
![image](https://github.com/johannes-wolf/cetz/assets/6866008/4281974e-cc1e-4088-ac75-315722c2deb9)
Intuitive way generates this:
```typst
line((0, 0), (0, 1), stroke: (thickness: .5pt, dash: "dashed"), mark: (end: ">"))
```
![image](https://github.com/johannes-wolf/cetz/assets/6866008/ac5367d3-585d-4a1d-93e0-021b08ad692a)

**Comment by johannes-wolf** [comment]
Mark could have a default style like `(paint: auto, dash: "solid", ...)`, which would disable inheriting the dash pattern by default.

**Comment by dscso** [comment]
I also see why it is implemented as it is @fenjalien, I just noticed that this might be a hustle when drawing images with the library as I did. I used the spreading operator to do something like that:
```typst
#let double_arrow = (mark : (end: ">", start: ">"))
#let single_arrow = (mark : (end: ">"))
#let dashed_arrow = (stroke: (dash: "dashed", thickness: .5pt), mark: (stroke: black, end: ">"))
...
```
and then
```typst
line("manager.west", "tcp.east", ..dashed_arrow, name: "manager_tcp")
```
and I thought this might be a bit verbose and would create problems if I wanted to use a dashed double arrow... Therefore I'd prefere the solution of @johannes-wolf I think this would fix the issue.

**Comment by johannes-wolf** [comment]
There is a style bug in 0.1.2, but something like `set-style(mark: (stroke: (dash: "solid", paint: auto)))` should work. The problem on 0.1.2 is, that the paint won't get inherited.
If you are using the same color for all your marks it should work though.

```typst
#canvas(length: 1cm, {
  import draw: *

  set-style(line: (mark: (stroke: (dash: "solid", paint: black))))
  line((0,0), (5,0), mark: (end: ">"), stroke: (dash: "dotted"))
})
```

**Comment by fenjalien** [comment]
I think this is solved as we can't reproduce this in 0.2.2, we also don't remember what the problem was or when we fixed it :)
